pom.xml

```diff
@@ -98,7 +98,7 @@
       <dependency>
         <groupId>org.codenarc</groupId>
         <artifactId>CodeNarc</artifactId>
-        <version>2.2.0</version>
+        <version>3.3.0</version>
```

Comment:
When updating CodeNarc, you should update the external at the same time and use the codenarc-converter to integrate changes in codenarc rules into this plugin

Reply:
Thanks for the quick response Tobi, i will do so.

Reply:
Done @TobiX , the build seems to go through.